api/docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Setting up the API container
+
+### Install docker and dependencies
+- Install Docker following the [official docs](https://docs.docker.com/get-docker/).
+
+### Build the container image
+- Ensure you're in `enimapod/api/`
+- Run `sudo docker build -f docker/Dockerfile -t enimapod-api .`
+
+### Run the container
+- Start the container, removing any existing container - `sudo docker rm -f enimapod-api; sudo docker run --name enimapod-api -p 8080:80 -d enimapod-api`
```

Comment:
```suggestion
- Start the container, removing any existing container - `sudo docker rm -f enimapod-api; sudo docker run --name enimapod-api -p 8081:8081 -d enimapod-api`
```

Reply:
It's `<hostPort>:<containerPort>`, so it was listening to port 80 on the container.